providers/register/provider_openstack.go

```diff
@@ -0,0 +1,19 @@
+package register
```

Comment:
Can you add a build tag to this?

```suggestion
// +build !no_openstack_provider

package register
```